life_hc/MySQL/Vehicles/fn_vehicleStore.sqf

```diff
@@ -2,44 +2,49 @@
 /*
     File: fn_vehicleStore.sqf
     Author: Bryan "Tonic" Boardwine
-
-    This file is for Nanou's HeadlessClient.
-
     Description:
     Stores the vehicle in the 'Garage'
 */
-private ["_vehicle","_impound","_vInfo","_vInfo","_plate","_uid","_query","_sql","_unit","_trunk","_vehItems","_vehMags","_vehWeapons","_vehBackpacks","_cargo","_saveItems","_storetext","_resourceItems","_fuel","_damage","_itemList","_totalweight","_weight"];
-_vehicle = [_this,0,objNull,[objNull]] call BIS_fnc_param;
-_impound = [_this,1,false,[true]] call BIS_fnc_param;
-_unit = [_this,2,objNull,[objNull]] call BIS_fnc_param;
-_storetext = [_this,3,"",[""]] call BIS_fnc_param;
-_ownerID = _unit getVariable ["life_clientID",-1];
-_resourceItems = LIFE_SETTINGS(getArray,"save_vehicle_items");
-
-if (isNull _vehicle || isNull _unit) exitWith {life_impound_inuse = false; _ownerID publicVariableClient "life_impound_inuse";life_garage_store = false;_ownerID publicVariableClient "life_garage_store";}; //Bad data passed.
-_vInfo = _vehicle getVariable ["dbInfo",[]];
-
-if (count _vInfo > 0) then {
+
+params [
+    ["_vehicle", objNull, [objNull]],
+    ["_impound", false, [true]],
+    ["_unit", objNull, [objNull]],
+    ["_storetext", "", [""]]
+];
+
+private _resourceItems = LIFE_SETTINGS(getArray,"save_vehicle_items");
+private _ownerID = _unit getVariable ["life_clientID",-1];
+
+if (isNull _vehicle || {isNull _unit}) exitWith {life_impound_inuse = false; _ownerID publicVariableClient "life_impound_inuse";life_garage_store = false;_ownerID publicVariableClient "life_garage_store";}; //Bad data passed.
+private _vInfo = _vehicle getVariable ["dbInfo", []];
+private "_plate";
+private "_uid";
+
+if !(_vInfo isEqualTo []) then {
     _plate = _vInfo select 1;
     _uid = _vInfo select 0;
 };
 
 // save damage.
+private "_damage";
```

Comment:
private _damage = if () then {} else {}
Or in this case even just `private _damage = []; if ()...`
and get rid of the else.